src/MetaVariables.cpp

```diff
@@ -131,29 +131,15 @@ void MetaVariables::subtractQueryFromPathInfo(std::string &pathInfo, const std::
 
 void MetaVariables::HTTPRequestToMetaVars(const HTTPRequest &request, MetaVariables &env)
 {
-	// TODO: several metavars have to be set from configfile
-
-	// This line will have the code put the interpreter path as argv[0] to execve
-	//  env.setVar("X_INTERPRETER_PATH", "/home/lmangall/.brew/bin/python3"); // school computer...
 	env.setVar("X_INTERPRETER_PATH", X_INTERPRETER_PATH);
-
```

Comment:
Since i'm reading this. Even if it's part of another Issue, you can check if when we set cgi_path this meta variable is set correctly and if execve is set with the right stuff.